cfpb/cfgov-refresh
# Simplify beta banner

Beta banner is way over-engineered. This removes significant JS, CSS, and markup. Fixes https://[GHE]/CFGOV/platform/issues/1899

## Removals

- Removes beta banner JS and associated unit test.
- Removes beta banner expandable-specific less.
- Removes beta banner expandable markup.

## Testing

1. Turn the beta banner flag on in wagtail.
2. visit site and observe non-expandable beta banner.

## Screenshots

<img width="1189" alt="screen shot 2017-10-27 at 1 49 13 pm" src="https://user-images.githubusercontent.com/704760/32117822-a63f5e0e-bb1d-11e7-994b-5ebff55e7d06.png">
